react-exercise/src/components/user-table/UserTable.scss

```diff
@@ -8,19 +31,45 @@
   }
 
   td {
-    padding: 0;
+    padding: 12px;
+    font-family: "Brown LL", sans-serif;
+    font-size: 14px;
+    color: var(--semantic-fg-neutral, #161719); 
+    line-height: 1.5;
+    vertical-align: middle;
   }
 
   th {
     padding: 10px;
-    background-color: #f7f7f7; // Background for all th elements
     font-weight: 600;
     color: var(--semantic-fg-primary, var(--semantic-fg-primary, #161719));
-    font-family: "Brown LL";
+    font-family: "Brown LL", sans-serif;
     font-size: 14px;
     font-style: normal;
     font-weight: 500;
     line-height: 18px;
     letter-spacing: -0.5px;
+
+    /* Sticky header */
+    position: sticky;
+    top: 0;
+    background-color: #f7f7f7; // Explicitly define a background color so when it scrolls it doesn't disappear
+    z-index: 1; /* Ensure the header is on top of table rows while scrolling */
+  }
+
+  .user-row {
+    background-color: #fff;
+    &:hover {
+      background-color: #f0f0f0;
+    }
+  }
+
+  .identifier-header {
+    width: 25%;
```

Comment:
Why do you need it to be 25% ? This could be problematic if you add a column for example.
Can you solve this with setting flex:1 for all the header cells ?

Reply:
I used flex boxes in the start, but this what made the problematic misalignment I had between the table header and the table data rows. When reading a bit I saw it could be solved if I will not use flex display.